tests/end2end/data/fileselect/single_fileselect.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
I wonder if it makes sense for this to be in two separate files? Might be easier to have two file selects in the same HTML file, so that you also can share some of the JS code around it?

Reply:
Yep, I'll try to make a single file :+1:

Reply:
Done